include/odp/api/spec/ipsec.h

```diff
@@ -569,6 +569,18 @@ typedef enum odp_ipsec_pipeline_t {
 } odp_ipsec_pipeline_t;
 
 /**
+ * IPSEC header type
+ */
+typedef enum odp_ipsec_ip_version_t {
+	/** Header is IPv4 */
+	ODP_IPSEC_IPV4 = 0,
+
+	/** Header is IPv6 */
+	ODP_IPSEC_IPV6
```

Comment:
@Bill-Fischofer-Linaro why would we like to have exact values for those enum names?